2023-12-26T00:23:39.0974555Z ##[group]Run actions/setup-python@v4
2023-12-26T00:23:39.0975141Z with:
2023-12-26T00:23:39.0975466Z   python-version: 3.8
2023-12-26T00:23:39.0975809Z   architecture: x64
2023-12-26T00:23:39.0976236Z   cache: pip
2023-12-26T00:23:39.0976566Z   check-latest: false
2023-12-26T00:23:39.0977105Z   token: ***
2023-12-26T00:23:39.0977479Z   update-environment: true
2023-12-26T00:23:39.0977875Z   allow-prereleases: false
2023-12-26T00:23:39.0978815Z ##[endgroup]
2023-12-26T00:23:39.2324623Z ##[group]Installed versions
2023-12-26T00:23:39.2400099Z Successfully set up CPython (3.8.18)
2023-12-26T00:23:39.2401676Z ##[endgroup]
2023-12-26T00:23:40.1278444Z [command]/opt/hostedtoolcache/Python/3.8.18/x64/bin/pip cache dir
2023-12-26T00:23:41.0504954Z /home/runner/.cache/pip
2023-12-26T00:23:42.4267608Z Received 6245959 of 6245959 (100.0%), 7.3 MBs/sec
2023-12-26T00:23:42.4268685Z Cache Size: ~6 MB (6245959 B)
2023-12-26T00:23:42.4296711Z [command]/usr/bin/tar -xf /home/runner/work/_temp/20e6b6fc-5add-45e3-a77b-7ca155aa69aa/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-12-26T00:23:42.4672008Z Cache restored successfully
2023-12-26T00:23:42.4693530Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.18-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
